apps/src/templates/projects/projectsRedux.js

```diff
@@ -1,12 +1,15 @@
 /** @file Redux actions and reducer for the Projects Gallery */
-import {combineReducers} from 'redux';
+
 import $ from 'jquery';
 import _ from 'lodash';
+import {combineReducers} from 'redux';
+
 import {Galleries} from './projectConstants';
 import {PUBLISH_SUCCESS} from './publishDialog/publishDialogRedux';
-import {DELETE_SUCCESS} from './deleteDialog/deleteProjectDialogRedux';
-import {channels as channelsApi} from '../../clientApi';
-import LibraryClientApi from '@cdo/apps/code-studio/components/libraries/LibraryClientApi';
+// Disabling import order due to circular dependencies errors from build.
```

Comment:
I would still like to see some more specificity here about what exactly the circular dependency is. One purpose for comments like this is to help future developers identify when the exception to the norm we've introduced could be removed.